modules/parrableIdSystem.js

```diff
@@ -203,9 +216,13 @@ function fetchId(configParams, gdprConsentData) {
     trackers,
     url: refererInfo.referer,
     prebidVersion: '$prebid.version$',
-    isIframe: utils.inIframe()
+    isIframe: utils.inIframe(),
   };
 
+  if (Date.now() < tpcUntil) {
+    data.tpcSupport = tpcSupport;
+  }
```

Comment:
I'm thinking this can be handled in readCookie(); so it doesn't return a tpcSupport value if the tpcUntil date has passed.

Reply:
In the readCookie() method we don't have knowledge of the property names, we just parse them and return an object. I don't think we should start adding this kind of logic there.